go/teams/create.go

```diff
@@ -10,16 +10,86 @@ import (
 	jsonw "github.com/keybase/go-jsonw"
 )
 
-func CreateRootTeam(ctx context.Context, g *libkb.GlobalContext, name string) (err error) {
-	defer g.CTrace(ctx, "CreateRootTeam", func() error { return err })()
+func CreateImplicitTeam(ctx context.Context, g *libkb.GlobalContext, impTeam keybase1.ImplicitTeamName) (res keybase1.TeamID, err error) {
+	defer g.CTrace(ctx, "CreateImplicitTeam", func() error { return err })()
 
-	g.Log.CDebugf(ctx, "CreateRootTeam load me")
-	me, err := libkb.LoadMe(libkb.NewLoadUserArg(g))
+	name, err := NewImplicitTeamName()
 	if err != nil {
-		return err
+		return res, err
 	}
+	teamID := RootTeamIDFromNameString(name)
+
+	// Load all the Keybase users
+	me, err := libkb.LoadMe(libkb.NewLoadUserArg(g))
+	if err != nil {
+		return res, err
+	}
+	var kbusers []*libkb.User
+	for _, kbu := range impTeam.KeybaseUsers {
+		var user *libkb.User
+		if kbu == me.GetName() {
+			user = me
+		} else {
+			user, err = libkb.LoadUser(libkb.LoadUserArg{
```

Comment:
that was likely a misstep on behalf of the earlier author. it will get painful for lots of users to not hit the cache here.  can a UPAK2 give you all you need?